app/components/three-way-compare-link.hbs

```diff
@@ -1,14 +1,14 @@
 {{#if this.visible }}
-  <AuLink
+  <AuButton
```

Comment:
This is unrelated to the bug we are fixing, but we shouldn't use links as buttons simply because they look like links.

Reply:
Hmm, it seems they were using a link to sidestep some styling conflicts with the form css hacks when using a button. A comment about this decision would have been nice :smile: